tests/integration/test_benchmark.py

```diff
@@ -672,7 +698,8 @@ def test_benchmark_single_table_with_output_destination(tmp_path):
         f'{output_destination}/SDGym_results_{today_date}/results_{today_date}_1.csv'
     )
     pd.testing.assert_frame_equal(results, saved_result, check_dtype=False)
-    pd.testing.assert_frame_equal(results, score_saved_separately, check_dtype=False)
+    results_no_adjusted = results.drop(columns=['Adjusted_Total_Time', 'Adjusted_Quality_Score'])
```

Comment:
Should the `score_saved_separately` also include the `Adjusted_Total_Time` and `Adjusted_Quality_Score`? I think it works if the very first step of the benchmark is to run the UniformSynthesizer, then when any synthesizer is saved, even independently, it could have those two columns.